### ISSUE TITLE
Problem importing pathway as pathway in cytoscape 3.7.1

### ISSUE THREAD

**Issue by IreneHemel** [wikipathways/cytoscape-wikipathways-app#106]
For my internship with @DeniseSl22 I want to import several pathways into cytoscape as a pathway, however when I try this on my computer (Windows 10 version 1803) in cytoscape 3.7.1 with version 3.3.7 of the WikiPathways app all the nodes are stacked.

![image](https://user-images.githubusercontent.com/49555069/57693969-ca960e80-764a-11e9-9e30-229040246356.png)
Image is an example based on WP4225, but it is the same for all pathways I tried

When I try the same thing in cytoscape 3.6.1, with the same Wikipathways app version, it does work, but not in 3.7.1
![image](https://user-images.githubusercontent.com/49555069/57695527-c9ff7700-764e-11e9-955f-d42cc3c797cc.png)

**Comment by mkutmon** [comment]
@IreneHemel I just tried it with 3.7.1 and app version 3.3.7 and it worked fine. Can you retry after restarting Cytoscape?

**Comment by IreneHemel** [comment]
When I restarted it yesterday and earlier today it didn't work (I worked in a saved session with other pathways that were imported as a network). I just tried it in a new session and then it does work, however when I open a pathway as a network and try to open one as a pathway after, the nodes are stacked again

**Comment by mkutmon** [comment]
ah okay, so it happens if you have multiple pathways open in one session? Does this also happen for other pathways?

**Comment by IreneHemel** [comment]
Yes, as soon as I open 1 pathway as a network I cannot open any as a pathway. So far this has happend for all the pathways I tried to open (WP4225, 4571, 4583, 4584 and 4595)

**Comment by AdamStuart** [comment]
Irene,

Sorry, I can’t reproduce it here.  It sounds like some variable is getting stuck in the wrong state.  You can probably uninstall and reinstall the app to clear it (faster than a restart), but it that’s not a very satisfying solution.

Check your default layout setting.  After importing as a network, Cytoscape would apply a layout, and maybe it is encountering a problem there.  For the sake of testing, I’d set Grid layout to be my default.  That will at least spread them out from a single spot.

I assume you are doing this via the user interface, as opposed to sending CyRest commands.  If you are using commands then please send me the script.

Adam

**Comment by IreneHemel** [comment]
My default layout is set to perfused force directed layout, when I change it to a grid layout the problem still appears. I can apply the layout after the stacked nodes appear and then it does apply the layout, however I cannot go back from that to the pathway layout I want. 

Disabeling and enabeling the app does work, so I will use that for now and I indeed work with the user interface, not with commands

**Comment by AlexanderPico** [comment]
This same thing happened to me yesterday. I could consistently reproduce it with these steps:

1. Fresh launch of Cytoscape **_(must be a fresh start; not just a "close session")_**
2. Search for WP179
3. Open as network
4. Repeat search for WP179
5. Open as pathway

Obs: overlapping nodes
Exp: pathway layout

Same thing happens when steps 2-5 are done with automation calls:

2. 'wikipathways import-as-network id=WP179'
3. 'wikipathways import-as-pathway id=WP179'

Once this has occured, then all pathway imports are affected, even upon restart. The only fix is to disable/enable the WP app.

**Comment by AdamStuart** [comment]
I was able to "fix" this by changing the declaration of the BasicVizPropStore class in GpmlToPathway.   (see Line 758)  It was previously static, but now we re-declare the objects with each instance.  Interface removed.

Note that BasicVizTableStore has the same static declaration.

**Comment by yihangx** [comment]
Reopen because of 3.3.9.

**Comment by AlexanderPico** [comment]
Fixed in 3.3.10

**Comment by BrandonMurugan** [comment]
I'm currently having this issue with WP v3.3.10 in Cytoscape 3.9.1; 3.10.0; and 3.10.1
It stacks the nodes when I open another WP pathway, and if I save and reopen a session file with just one WP pathway loaded

**Comment by BrandonMurugan** [comment]
Hi there. So I did figure out what happened. When I load a WP pathway, it loads two Views. One is formatted properly, and the other one is stacked and has the KGML_style applied. Deleting the offending View fixes it. The appearance of the "2" in the network list made me suspicious. See attachments below. 
It does seem to be a Cytoscape issue, since it does this for iGraph imports from R.
I have no idea if it's just happening to me (maybe some setting/app) or is a more widespread issue.

![network_list](https://github.com/wikipathways/cytoscape-wikipathways-app/assets/23439986/b35e6996-c0fd-460d-bc76-1e0045e6079c)
![grid_view](https://github.com/wikipathways/cytoscape-wikipathways-app/assets/23439986/adbfa3a5-0e8e-4081-ae22-f7182c92c89f)

Cheers!

**Comment by AlexanderPico** [comment]
Wow. That's interesting. Haven't seen that before. You mention the "KGML_style". That's coming from a KEGG app, I presume. Have you tried disabling that (and other apps)? Sometimes apps can collide.
